### PR TITLE
Fix phpDoc.

### PR DESCRIPTION
**Pull request by Metalaka**

### CODE REVIEW

File: Client.php [hunk 1]
```diff
@@ -216,8 +216,9 @@ public function select()
      *
      * @param   \Hoa\Socket\Connection  $other    Other client.
      * @return  \Hoa\Socket\Client
+     * @throws  \Hoa\Core\Exception
      */
-    public function consider(Connection $other)
+    public function consider(Connection\Connection $other)
```
Comment (Hywan) [inline Client.php]: This is not needed since this is a flex-entity. I guess you can revert them.

**Comment by Hywan** [comment]
Except the fact I prefer to not unfold flex-entity names, everything is OK!

**Comment by Metalaka** [comment]
Rebased.

**Comment by Hywan** [comment]
Thanks!

**Pull request merged**